message/migrations/0002_test_data.py

```diff
@@ -0,0 +1,57 @@
+# Generated by Django 4.0.3 on 2022-03-30 19:27
+
+from django.db import migrations, transaction
+
+EMAIL = 'TESTTESTuser12513@gmail.com'
+USERNAME = 'testuserTESTEST'
+PASSWORD = 'testpassowrd'
+DOG_NAME = 'kliford'
+FIRST_NAME = 'NEWTESTETS'
+LAST_NAME = 'USERTESTEST'
+PHONE_NUMBER = 1234567890
+DOG_RACE = 'lavrador'
+DOG_PICTURE_URL = 'https://www.google.com/'
+DOG_AGE = 10
+DOG_WEIGHT = 6
+DOG_GENDER = 'M'
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('message', '0001_initial'),
+        ('dogowner', '0002_test_data'),
+    ]
+
+    def generate_data(apps, schema_editor):
+        from message.models import Message
+        from dogowner.models import DogOwner
+
+        test_data = [
+           ('Author1', '54321', 'First message'),
+           ('Author2', '9876', 'Second message'),
+        ]
+
+        with transaction.atomic():
+            dogonwer = DogOwner.create(email=EMAIL,
```

Comment:
you can use the DogOwner that was created in the data migrations of the DogOwner 
https://github.com/redhat-beyond/Hotails/blob/main/dogowner/migrations/0002_test_data.py